docs/routing/special_routes.md

```diff
@@ -0,0 +1,62 @@
+# Special routes
+
+To see special routes, use the `Show special` switcher at the top of the routes page
+
+It is used to render errors, if an application with the `Primary` or `Essential` type falls on the page, ILC will not find a match between the address bar and `Route`, `Template` will not be specified in the route chain, etc., ILC will use `Special route` or simple template with an error, depending on the situation:
+
+## '404' Error
+
+![ILC registry special routes switcher](../assets/routes/special-routes-switcher.png)
+
+- This error usually means that the requested resource was not found on the server.
```

Comment:
What do you mean?
404 is rendered only if there is no matched route in ILC Registry.